app/adapters/user.js

```diff
@@ -14,6 +14,6 @@ export default V3Adapter.extend({
   // This overrides the parent implementation to ignore the query parameters
   queryRecord(store, type, query) {
     let url = this.buildURL(type.modelName, null, null, 'queryRecord', query);
-    return this.ajax(url, 'GET', { data: {include: 'owner.installation' }});
+    return this.ajax(url, 'GET', { data: {include: query.included}});
```

Comment:
I like this parameterisation of `include`… in previous work I’ve added yet another item to the `include`, which has felt wasteful to me because so many things are being returned for all possible queries. If I need to do this again, I’ll be able to do something like this to only include things when they’re needed, so thanks!